execution_workflows/GETUTR/workflow/execute_getutr.nf

```diff
@@ -0,0 +1,56 @@
+////////////////////////////////////////////////////
+/* --         LOCAL PARAMETER VALUES           -- */
+////////////////////////////////////////////////////
+
+params.summary_params = [:]
+
+////////////////////////////////////////////////////
+/* --          VALIDATE INPUTS                 -- */
+////////////////////////////////////////////////////
+
+// Check input path parameters to see if they exist
+
+checkPathParamList = [ params.input ]
+for (param in checkPathParamList) { if (param) { file(param, checkIfExists: true) } }
+
+// Check mandatory parameters (missing protocol or profile will exit the run.)
+
+if (params.input) { ch_input = file(params.input) } else { exit 1, 'Input samplesheet not specified!' }
+//if (params.labrat_quant_dir) { ch_labrat_quant_dir = file(params.labrat_quant_dir) } else { ch_labrat_quant_dir = '' }
```

Comment:
to remove

Reply:
Sorry what should we remove above?

Reply:
yes right, adding also here the gtf param, thanks!